modules/porous_flow/doc/content/source/actions/PorousFlowAddBCAction.md

```diff
@@ -0,0 +1,25 @@
+# PorousFlowAddBCAction
+
+This action creates `porous_flow`-specific boundary conditions. It allow developers to create
+user friendly syntax for adding more complicated boundary conditions using the following structure:
+
+```
+[PorousFlow]
+  [BCs]
+    [my_bc]
+      type = MyUserFriendlyBC
+      parameters
+    []
+  []
+[]
+```
+
+To create a syntax like, this developers have to build a proxy class called `MyUserFriendlyBC`.
```

Comment:
This sentence doesn't quite make sense

Reply:
because of the traveling comma :-), should be:

> To create a syntax like this, developers have to build a proxy class called `MyUserFriendlyBC`.